styles.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Vesper+Libre:wght@500&display=swap');
+*, *::before, *::after {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    
+}
+#body {
+    min-width: 290px;
+    
+    background-color: #a2a1a1;
+    font-family: 'Vesper Libre', serif;
+    line-height: 1.5;
+  }
+  #navbar {
+    position: fixed;
+    min-width: 290px;
+    top: 0px;
+    left: 0px;
+    width: 300px;
+    height: 100%;
+    background-color:  #4e21c1;
+    color: #ffffff;
+    border-right: solid;
+    border-color: rgba(0, 22, 22, 0.4);
+  }
+  #navbar header {
+    color: #4e21c1;
+    background-color: rgb(216, 196, 45);
+    text-align: center;
+    font-size: 2.5em;
+    font-family: 'Vesper Libre', serif;
+    font-weight: thin;
+  }
+  #navbar li {
+    color: #ffffff;
+    border-top: 1px solid;
+    list-style: none;
+    position: relative;
+    width: 100%;
+  }
+  #navbar a {
+    display: block;
+    padding: 10px 30px;
+    color:rgb(255, 255, 255);
+    text-decoration: none;
+    cursor: pointer;
+  }
+#main-doc {
+    position: absolute;
+    margin-left: 300px;
+    padding: 20px;
+    margin-bottom: 110px;
+    background-color:rgb(215, 215, 215)
+  }
+#main-doc header {
+    color: rgb(0, 0, 0);
+    margin: 30px;
+    text-align: left;
+    font-size: 1.8em;
+    font-weight: bold;
+  }
+#main-doc code {
+    display: block;
+    text-align: left;
+    white-space: pre-line;
+    position: relative;
+    word-break: normal;
+    word-wrap: normal;
+    line-height: 2;
+    background-color: #f7f7f7;
+    padding: 15px;
+    margin: 10px;
+    border-radius: 5px;
+  }
+  @media only screen and (max-width: 400px) {
+    #main-doc {
+      margin-left: -10px;
+    }
+  
+    code {
+      margin-left: -20px;
+      width: 100%;
+      padding: 15px;
+      padding-left: 10px;
+      padding-right: 45px;
+      min-width: 233px;
+    }
+  }
```

Comment:
- [ ] [REQUIRED] As you can see, in line 89, you have a bracket to spare. You should erase it and that's it!